### PR TITLE
Add support for Mac & Linux + InternalWebBrowser

### PR DESCRIPTION
**Pull request by dirtycajunrice**
* Updated readme with more descriptive information
* Moved from the self-hosted http flow to an internal WebBrowser flow
* Added additional blueprint utility functions
* custom WebBrowser widget and userwidget
* Additional binaries added for next platform support stage

### CODE REVIEW

**Comment by 0xFirekeeper** [comment]
Apple login appears to have a transparent background now, might need extra options? Believe it wasn't like this in initial tests
745c8d092c36dd0400801b1f260df7bf

File: Thirdweb.uplugin [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+	"Installed": false,
+	"Modules": [
+		{
+			"Name": "Thirdweb",
+			"Type": "Runtime",
+			"LoadingPhase": "PreDefault",
+			"PlatformAllowList": [
+				"Win64",
+				"Mac",
+				"Linux"
```
Comment (0xFirekeeper) [inline Thirdweb.uplugin:25]: Should LinuxArm64 be added here?

Comment (0xFirekeeper) [reply Thirdweb.uplugin:25]: Also, should the mobile static libs be removed from the package for the time being?

**Comment by 0xFirekeeper** [comment]
Unimportant: bunch of TCHAR_TO_UTF8 left
Otherwise looking good overall, good work!

File: Thirdweb.uplugin (hunk 1 above)
Comment (dirtycajunrice) [reply Thirdweb.uplugin:25]: Needs testing then maybe

**Pull request merged**